docs/EN_US/DataPatterns/DataPatternsMods/DataPatternsInECLWatch.xml

```diff
@@ -1,18 +1,178 @@
+
+    <listitem>
+      <para>Click on the title “Data Patterns” on the bar located near the top
+      of the page. A report of the data file is generated.</para>
+
+      <graphic fileref="../../images/DP003.jpg"/>
+    </listitem>
+  </orderedlist>
+
+  <para>Every attribute in the Profile result is represented by a row of
+  information. Each row of information is organized into several columns. Here
+  is a short description of each column:</para>
+
+  <itemizedlist>
+    <listitem>
+      <para>Type information, Cardinality Count &amp; Filled Count</para>
+    </listitem>
+
+    <listitem>
+      <para>Min, Avg, Max Length (for string attributes) or Mean, Std.
+      Deviation, Quartiles (for numeric attributes)</para>
+    </listitem>
+
+    <listitem>
+      <para>Quartile bell curve and candlestick</para>
+
+      <itemizedlist>
+        <listitem>
+          <para>only shown for attributes with is_numeric === true</para>
+        </listitem>
+
+        <listitem>
+          <para>this column is omitted if the above condition fails for all
+          attributes</para>
+        </listitem>
+      </itemizedlist>
+    </listitem>
+
+    <listitem>
+      <para>Cardinality Breakdown listed by count descending</para>
+
+      <itemizedlist>
+        <listitem>
+          <para>only shown for attributes with cardinality_breakdown
+          content</para>
+        </listitem>
+
+        <listitem>
+          <para>this column is omitted if the above condition fails for all
+          attributes</para>
+        </listitem>
+      </itemizedlist>
+    </listitem>
+
+    <listitem>
+      <para>Popular Patterns</para>
+
+      <itemizedlist>
+        <listitem>
+          <para>only shown for attributes with popular_patterns content</para>
+        </listitem>
+
+        <listitem>
+          <para>this column is omitted if the above condition fails for all
+          attributes</para>
+        </listitem>
+      </itemizedlist>
+    </listitem>
+  </itemizedlist>
+
+  <para>For example:</para>
+
+  <graphic fileref="../../images/DP004.jpg"/>
+
+  <para>As seen in the “state” section, the cardinality of each state is
+  displayed. The cardinality breakdown is shown for attributes with a low
+  number of unique, non-nil values, showing each value and the number of
+  records that contain the value.</para>
+
+  <para><emphasis role="bold">Note</emphasis>: The lcbLimit parameter governs
+  what “low number” means</para>
+
+  <para>In this example, the cardinality for “NY” is 7%, meaning that 7% of
+  records contain the value “NY”.</para>
+
+  <para>The “Min Length'' displays the shortest length of a value, where null
+  values are ignored. In SET datatypes, however, “Min Length'' may display the
+  fewest number of elements in the set.</para>
+
+  <para>The “Max Length'' displays the longest length of a value, where null
+  values are also ignored. The largest number of elements in the set will be
+  displayed for SET datatypes.</para>
```

Comment:
will be s/b is
The largest number of elements in the set **is** displayed for SET datatypes.